ex1/exceptions.py

```diff
@@ -37,6 +37,11 @@
 # Use the function search_n from the functions module inside a new function
 # also named search_n. The function should do the same as functions.search_n
 # but if the variable is not found in the list then raise a ValueError.
+def search_n(ls, x):
+	if functions.search_n(ls, x) == (None, None):
```

Comment:
Using `==` when comparing with Python Singletons is not recommended.
If you are interested why that is then please see the answers [here](http://stackoverflow.com/questions/2209755/python-operation-vs-is-not)